gtk/CMakeLists.txt

```diff
@@ -158,24 +158,16 @@ target_compile_definitions(${TR_NAME}-gtk
     PRIVATE
         TRANSMISSIONLOCALEDIR="${CMAKE_INSTALL_FULL_LOCALEDIR}"
         GETTEXT_PACKAGE="${TR_NAME}-gtk"
-        # FIXME: these break libnotify's headers
-        # G_DISABLE_SINGLE_INCLUDES
-        # GTK_DISABLE_SINGLE_INCLUDES
         $<$<BOOL:${ENABLE_UTP}>:WITH_UTP>)
 
-if(DISABLE_DEPRECATED)
+if(NOT ENABLE_DEPRECATED)
 	target_compile_definitions(${TR_NAME}-gtk
```

Comment:
Please use spaces for indentation. Tabs may have different width depending on local setup.